README.md

```diff
@@ -135,6 +157,33 @@ setupController: function(controller, model) {
   controller.set('products', model);
 }
 ```
+### event hooks
+
+ember-infinity provides a route mixin that enables us to hook into the following events:
```

Comment:
It's implied the user is already using the route mixin, we can just say:

"The route mixin also provides the following event hooks:"